ddpui/api/charts_api.py

```
@@ -787,86 +787,100 @@ def generate_map_chart_data(request, payload: ChartDataPayload):
 
-                for row in data:
-                    writer.writerow(row)
+        output.close()
 
-                # Yield current chunk
-                yield output.getvalue()
-                output.truncate(0)
-                output.seek(0)
+    except Exception as error:
+        logger.exception(
+            f"Error streaming chart data for schema {payload.schema_name}.{payload.table_name}: {str(error)}"
```

Ruff (0.14.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)